src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1370,6 +1370,9 @@
   <data name="EnumShouldNotHaveDuplicatedValuesTitle" xml:space="preserve">
     <value>Enums values should not be duplicated</value>
   </data>
+  <data name="DoNotIgnoreMethodResultsMessageLinqMethod" xml:space="preserve">
+    <value>{0} calls {1} but does not use the value the method returns. Linq methods are known to not have side effects. Use the result in a conditional statement, assign the result to a variable, or pass it as an argument to another method.</value>
```

Comment:
```suggestion
    <value>'{0}' calls '{1}' but does not use the value the method returns. Linq methods are known to not have side effects. Use the result in a conditional statement, assign the result to a variable, or pass it as an argument to another method.</value>
```

Reply:
At some point it might be worthwhile to write an analyzer/fixer that catches and auto-fixes these - the resx file would be expected to be passed as an additional file...